Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -100,16 +101,16 @@ public RubberduckParserState(IVBE vbe)
 
         private void AddComponentEventHandlers(IVBProject project)
```

Comment:
Good catch. I probably should have cleaned it out a bit - I see I also left some commented out code in there too.